docs/chaos/index.md

```diff
@@ -0,0 +1,53 @@
+# Chaos engineering with Simmy
+
+[Simmy][simmy] is a major new companion project adding a chaos-engineering and fault-injection dimension to Polly, through the provision of policies to selectively inject faults, latency, custom behavior or fake results.
+
+<img src="../../logos/Simmy-Logo.png" alt="Simmy"/>
+
+## Motivation
+
+There are a lot of questions when it comes to chaos-engineering and making sure that a system is actually ready to face the worst possible scenarios:
+
+* Is my system resilient enough?
+* Am I handling the right exceptions/scenarios?
+* How will my system behave if X happens?
+* How can I test without waiting for a handled (or even unhandled) exception to happen in my production environment?
+
+Using Polly helps me introduce resilience to my project, but I don't want to have to wait for expected or unexpected failures to test it out. My resilience could be wrongly implemented; testing the scenarios is not straight forward; and mocking failure of some dependencies (for example a cloud SaaS or PaaS service) is not always straightforward.
+
+**What do I need, to simulate chaotic scenarios in my production environment?**
+
+* A way to mock failures of dependencies (any service dependency for example).
+* Define when to fail based on some external factors - maybe global configuration or some rule.
+* A way to revert easily, to control the blast radius.
+* Production grade, to run this in a production or near-production system with automation.
+
+## Chaos strategies (a.k.a Monkey strategies)
+
+Chaos strategies (or Monkey strategies as we call them) are in essence a [Resilience strategy](../strategies/index.md#built-in-strategies), which means, as well as a *Resilience Strategy* is the minimum unit of resilience for Polly, a *Monkey Strategy* is the minimum unit of chaos for Simmy.
+
+### Built-in strategies
+
+|Strategy| Reactive| What does the policy do?|
+| ------------- |------------- |------------- |
+|[Fault](fault.md)|No|Injects exceptions in your system.|
+|[Result](result.md)|Yes|Substitute results to fake outcomes in your system.|
+|[Latency](latency.md)|No|Injects latency into executions before the calls are made.|
+|[Behavior](behavior.md)|No|Allows you to inject *any* extra behaviour, before a call is placed.|
```

Comment:
Can you make the spacing in the table a bit nicer to read like the others with spaces please?

Reply:
Done